ome/figure
# Xywh panel

Testing instructions : 

1) Try manually manipulating X,Y Width and Height values in the info panel (both single and multi-select cases).
Check if the changes get updated on the panels.
2) Try dragging an image from one page to another and check if the X,Y values are reset to 0,0 at the start of every new page.
3) Multi-select : Select 2 images from 2 different pages (1 image from page 1 and 1 image from page 2) and change X,Y values. Check if the images get aligned to the appropriate X,Y positions within the page that they reside in. (i.e : image 1 is set to X,Y in page 1 , and image 2 is set to the same X,Y in page 2)

@will-moore
